src/sentry/static/sentry/app/views/projects/projectContext.jsx

```diff
@@ -13,7 +13,7 @@ import withApi from 'app/utils/withApi';
 import LoadingError from 'app/components/loadingError';
 import LoadingIndicator from 'app/components/loadingIndicator';
 import MemberListStore from 'app/stores/memberListStore';
-import MissingProjectMembership from 'app/components/missingProjectMembership';
+import MissingProjectMembership from 'app/components/projects/missingProjectMembership';
```

Comment:
Should this be local to `projectContext`?

Reply:
I moved the 2 files into the same folder, but thought it wasn't necessary to nest too deeply